src/shared/components/person/settings.tsx

```diff
@@ -527,7 +525,7 @@ export class Settings extends Component<any, SettingsState> {
                   .sort((a, b) => a.code.localeCompare(b.code))
                   .map(lang => (
                     <option value={lang.code}>
-                      {getNativeLanguageName(lang.code)}
+                      {getLanguages(lang.code)[0]}
```

Comment:
Use the language name here, and I'm not sure why this needed to call getLanguages again.

Reply:
True, i changed this before merging the language lists.